src/app/modules/item/pages/item-by-id/item-by-id.component.ts

```diff
@@ -205,13 +183,12 @@ export class ItemByIdComponent implements OnDestroy, BeforeUnloadComponent, Pend
     }),
   );
 
-  contentContainerTop = 0;
-
   private subscriptions: Subscription[] = [
     this.itemDataSource.state$.pipe(
       // submission reloads the item data. Here we handle the progress tab existence only when the item loads, not when it reloads.
       distinctUntilChanged((a, b) => a.data?.route.id === b.data?.route.id),
     ).subscribe(state => {
+      this.discussionService.toggleVisibility(false);
```

Comment:
might conflict with my tab-related changes. If so, just move it under `itemChanged$` subscription.